src/Microsoft.SqlTools.ServiceLayer/Security/Contracts/LoginRequest.cs

```diff
@@ -62,7 +52,78 @@ public class DeleteLoginRequest
         /// Request definition
         /// </summary>
         public static readonly
-            RequestType<DeleteLoginParams, ResultStatus> Type =
-            RequestType<DeleteLoginParams, ResultStatus>.Create("security/deletelogin");
+            RequestType<DeleteLoginParams, object> Type =
+            RequestType<DeleteLoginParams, object>.Create("objectManagement/deleteLogin");
+    }
+
+    /// <summary>
+    /// Update Login params
+    /// </summary>
+    public class UpdateLoginParams : GeneralRequestDetails
+    {
+        public string ContextId { get; set; }
+
+        public LoginInfo Login { get; set; }
+    }
+
+    /// <summary>
+    /// Update Login request type
+    /// </summary>
+    public class UpdateLoginRequest
+    {
+        /// <summary>
+        /// Request definition
+        /// </summary>
+        public static readonly
+            RequestType<UpdateLoginParams, object> Type =
+            RequestType<UpdateLoginParams, object>.Create("objectManagement/updateLogin");
+    }
+
+
+    /// <summary>
+    /// Update Login params
+    /// </summary>
+    public class DisposeLoginViewRequestParams : GeneralRequestDetails
```

Comment:
probably don't need to derive from `GeneralRequestDetails`.  I think this to get the property bag support.